assistant/core.py

```
@@ -260,7 +260,11 @@ def get_completion_with_retry(self, messages: List[Dict[str, Any]] = None, max_r
                 else:
                     raise
         
-        raise Exception("Failed to get completion after maximum retries")
+        raise APICallError(
```

Ruff (0.8.2)

Undefined name `APICallError`

(F821)